kit/src/main/resources/com/oracle/javafx/scenebuilder/kit/i18n/SceneBuilderKit_zh_CN.properties

```diff
@@ -76,12 +76,12 @@ log.info.explore.jar.results = 探索 JAR {0} 的结果
 log.info.explore.folder.results = 探索文件夹 {0} 的结果
 log.info.explore.no.results = 未找到自定义控件
 log.warning.inline.edit.internationalized.strings = 无法内联编辑国际化字符串
-log.warning.color.creation.error.hexadecimal = 无法为十六进制值 “{0}” 创建颜色
-log.warning.image.location.does.not.exist = 图像 “{0}” 不存在
-log.warning.invalid.fxid = Id “{0}” 不是有效的 Java 标识符
-log.warning.no.injectable.fxid = 在 FXML 控制器类中找不到 id “{0}” 的可注入字段
-log.warning.duplicate.fxid = Id 为 “{0}” 的元素已注入 FXML 控制器
-log.warning.layout.failed = 布局失败 (“{0}”)
+log.warning.color.creation.error.hexadecimal = 无法为十六进制值 ''{0}'' 创建颜色
+log.warning.image.location.does.not.exist = 图像 ''{0}'' 不存在
+log.warning.invalid.fxid = Id ''{0}'' 不是有效的 Java 标识符
+log.warning.no.injectable.fxid = 在 FXML 控制器类中找不到 id ''{0}'' 的可注入字段
+log.warning.duplicate.fxid = Id 为 ''{0}'' 的元素已注入 FXML 控制器
+log.warning.layout.failed = 布局失败 (''{0}'')
```

Comment:
Can you please explain why is this change required?

Reply:
I think this is the correct format.

Reply:
I googled my way since I am not a Chinese expert and landed on this [Quora](https://www.quora.com/Should-Simplified-Chinese-use-Chinese-quotation-marks-instead-of-Western-ones/answer/Barron-Lee) and it links to a document which states that double quotes should be fine.

@tiainen since you know Chinese, do you have a preference?

Reply:
It has actually nothing to do with the Chinese language. Escaping single quotes (with a single quote) is required if you want to print the single quote. It works this way, because when the property value is parameterized, `java.text.MessageFormat` is used to format the string, which uses this type of escaping single quotes.

Reply:
@tiainen Thanks for the explanation. Escaping single quotes never crossed my mind and I kept thinking it has something to do with the language instead :)